src/views/blocks/tickets/quantity-remove.php

```diff
@@ -11,7 +11,9 @@
  * @link {INSERT_ARTICLE_LINK_HERE}
  *
  * @since 4.9.3
- * @version 4.11.0
+ * @since TBD Updated the button to include a type - helps avoid submitting forms unintentionally.
+ *
+ * @version TBD
  *
  */
```

Comment:
This one is missing the change - let me check the current code though. It might be missing?

Reply:
It's in https://github.com/moderntribe/event-tickets/commit/b955ca810db1f1b93316e764d1f96a3809ee344f

Reply:
Just added the change.